types/react-instantsearch/dom.d.ts

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+
+import { Hit, BasicDoc } from 'react-instantsearch-core';
+
+// Core
+export { createConnector } from 'react-instantsearch-core';
+export { HIGHLIGHT_TAGS } from 'react-instantsearch-core';
+export { translatable } from 'react-instantsearch-core';
+
+// Widget
+export { Configure } from 'react-instantsearch-core';
+
+// Connectors
+export { connectAutoComplete } from 'react-instantsearch-core';
```

Comment:
All these should be imported from 'react-instantsearch-dom' package instead. as per the dom.js in the package.